### DISCUSSION TITLE
Ergo :  Unable to identify network type

### DISCUSSION THREAD

**Discussion by makertronic** [coinfoundry/miningcore#1317] [General]
Hi i cant start Miningcore i got this message : 
[2022-07-17 13:14:10.1735] [E] [ergo1] Unable to identify network type (NodeErgo
[2022-07-17 13:14:10.1735] [E] [Core] Cluster cannot start. Good Bye! 

Any clue about this ? 

here is my node config : 
ergo {
    networkType = "mainnet"
    directory = "/home/nicola/ergo/"
    node {
        mining = true
        useExternalMiner = true
    }
    wallet.secretStorage.secretDir = "/home/nicola/ergo/wallet/keystore"
    voting {
        8 = 1000
    }
    chain {
        reemission {
        checkReemissionRules = true
        }
    }
    wallet {
        checkEIP27 = true
    }
}

scorex {

        network {
                nodeName = "NodeErgo"
                bindAddress = "0.0.0.0:9020"
        }

**Comment by oliverw** [comment]
For now get rid of the `nodeName` property. Fixed by c58a647b5eb9953ac6e1260ae63dcadf18c68506

**Reply by makertronic** [reply]
Hi !

Do i have to compile a new version ? master ?
